lib/localization/arb/intl_en.arb

```diff
@@ -714,6 +714,11 @@
     "type": "text",
     "placeholders": {}
   },
+  "useMediaStoreForFavoriteSongsSetting": "Use Android system for favorite tracks",
```

Comment:
This is misleading, because first, it doesn't synchronize it, it literally uses the Android MediaStore as data source, secondly, it does this only for tracks